index.bs

```diff
@@ -23,60 +23,26 @@ spec:html; type:dfn; text:port message queue
 
 <pre class="anchors">
 urlPrefix: https://tc39.es/ecma262/; spec: ECMASCRIPT
- type: constructor
-  text: %Uint8Array%; url: #sec-typedarray-objects
-  text: %DataView%; url: #sec-dataview-constructor
-  text: %ArrayBuffer%; url: #sec-arraybuffer-constructor
  type: interface
   text: ArrayBuffer; url: #sec-arraybuffer-objects
   text: DataView; url: #sec-dataview-objects
-  text: Number; url: #sec-ecmascript-language-types-number-type
   text: SharedArrayBuffer; url: #sec-sharedarraybuffer-objects
   text: Uint8Array; url: #sec-typedarray-objects
-  text: %Object.prototype%; url: #sec-properties-of-the-object-prototype-object
  type: dfn
   text: abstract operation; url: #sec-algorithm-conventions-abstract-operations
   text: array; url: #sec-array-objects
   text: async generator; url: #sec-asyncgenerator-objects
   text: async iterable; url: #sec-asynciterable-interface
-  text: completion record; url: #sec-completion-record-specification-type
   text: internal slot; url: #sec-object-internal-methods-and-internal-slots
-  text: iterable; url: #sec-iterable-interface
```

Comment:
This now autolinks to https://webidl.spec.whatwg.org/#dfn-iterable, where it used to link to https://tc39.es/ecma262/#sec-iterable-interface.

We probably need to override the link in https://streams.spec.whatwg.org/#rs-prototype to always use the definition from ECMASCRIPT rather than WEBIDL.